### PR TITLE
Feature/104 Implemented test for accounting service

### PR DESCRIPTION
**Pull request by nasaku898**
resolves #105

### CODE REVIEW

File: bike-erp/backend/src/main/services/accountingService/AccountingService.ts [hunk 1]
```diff
@@ -31,11 +36,9 @@ export class AccountingService {
         orderList.forEach((order) => {
             total = total + order.price * order.quantity;
         })
-
         try {
             // Creating account payable in db
             const accountPayableId = await AccountingService.accountPayableDAO?.createAccountPayable(total, new Date().toISOString().slice(0, 19).replace('T', ' '), email) as number;
-
```
Comment (Manraj45) [inline bike-erp/backend/src/main/services/accountingService/AccountingService.ts:38]: Readd white space here for clarity

File: bike-erp/backend/src/test/services/accountingService/accountingService.test.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import { AccountPayableDAO } from "../../../main/dao/AccountPayableDAO";
+import { AccountReceivableDAO } from "../../../main/dao/AccountReceivableDAO";
+import db from "../../../main/helpers/db";
+import { BikeOrder } from "../../../main/models/interfaces/BikeOrder";
+import { AccountingService, Order } from "../../../main/services/accountingService/AccountingService"
+describe("Accounting Service Test", () => {
```
Comment (Manraj45) [inline bike-erp/backend/src/test/services/accountingService/accountingService.test.ts:6]: Format unit test (add space) and add comments

File: bike-erp/backend/src/test/services/registrationService/registrationService.test.ts [hunk 3]
```diff
@@ -1,23 +1,24 @@
 import dotenv from "dotenv";
 import db from "../../../main/helpers/db";
 import { RegistrationService } from "../../../main/services/registrationService/RegistrationService";
-
```
Comment (Manraj45) [inline bike-erp/backend/src/test/services/registrationService/registrationService.test.ts:4]: add space

File: bike-erp/backend/src/main/services/accountingService/AccountingService.ts [hunk 4]
```diff
@@ -6,12 +6,10 @@ import { BikeOrder } from "../../models/interfaces/BikeOrder";
 export class AccountingService {
```
Comment (jmezaber) [inline bike-erp/backend/src/main/services/accountingService/AccountingService.ts:7]: Maybe add a header comment?

**Review by jmezaber — COMMENTED** [review]
Good job Simon!

**Review by Manraj45 — APPROVED** [review]
Good job!

**Review by jmezaber — APPROVED** [review]

**Pull request merged**